src/components/ColorProvider.tsx

```
@@ -29,28 +29,48 @@
+          toast.warning(
+            "The capitalization of this server was incorrect. If your using a permanent link resource, please change it to account for a new name. (" +
+              c.server.name +
+              ") Redirecting now.",
+            { duration: 15000 }
+          );
+          nav.replace("/server/" + c.server.name);
+        }
+      });
+  });
 
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: fetchV

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)